src/daemon.py

```diff
@@ -97,13 +97,8 @@ def job_daemon() -> None:
                 )
 
         elif queue == "queue-metadata":
-            # LEGACY QUEUE
```

Comment:
this reverts changes from other PR. You probably resolved merge conflicts by using "keep my changes only" instead of merging properly